docs/docs/docs/deployment/index.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: docs
+title: Deployment
+url: /docs/deployment
+---
+
+# Deployment
+
+## Requirements
+
+In order to deploy Compendium, it is necessary to provide an storage for both metadata and protocols. **Each one will use its own database** to store the necessary information.
+
+#### Supported storages for metadata:
+
+- PostgreSQL
+
+#### Supported storages for protocols:
+
+- PostgreSQL
+
+## Compendium configuration variables
+
+Compendium service offers multiple variables that allows user to configure how the service is deployed. All variables are defined at Compendium's [application.conf][applicationconf].
+
+#### Metadata Storage configuration:
+
+- `COMPENDIUM_METADATA_STORAGE_JDBC_URL`
+- `COMPENDIUM_METADATA_STORAGE_USERNAME`
+- `COMPENDIUM_METADATA_STORAGE_PASSWORD`
+- `COMPENDIUM_METADATA_STORAGE_CONNECTION_TIMEOUT`
+- `COMPENDIUM_METADATA_STORAGE_IDLE_TIMEOUT`
+- `COMPENDIUM_METADATA_STORAGE_MAX_LIFETIME`
+- `COMPENDIUM_METADATA_STORAGE_MINIMUM_IDLE`
+- `COMPENDIUM_METADATA_STORAGE_MAXIMUM_POOL_SIZE`
+
+#### Protocol Storage configuration:
+
+- `COMPENDIUM_PROTOCOLS_STORAGE_JDBC_URL`
+- `COMPENDIUM_PROTOCOLS_STORAGE_USERNAME`
+- `COMPENDIUM_PROTOCOLS_STORAGE_PASSWORD`
+- `COMPENDIUM_PROTOCOLS_STORAGE_CONNECTION_TIMEOUT`
+- `COMPENDIUM_PROTOCOLS_STORAGE_IDLE_TIMEOUT`
+- `COMPENDIUM_PROTOCOLS_STORAGE_MAX_LIFETIME`
+- `COMPENDIUM_PROTOCOLS_STORAGE_MINIMUM_IDLE`
+- `COMPENDIUM_PROTOCOLS_STORAGE_MAXIMUM_POOL_SIZE`
+
+[applicationconf]: https://github.com/higherkindness/compendium/blob/master/src/main/resources/application.conf
```

Comment:
In my experience as a newbie in compendium, I would  appreciate a small section about running it in local, so I can test it quickly before going further.

Reply:
Official `compendium` distribution, right now, is using Docker. 
If someone wants to give `compendium` a try, the fastest way is going to be running the official image, using `Docker` or `docker-compose` as I have documented in this PR.

Nonetheless, I agree we should add a new section: `Contributing`. This section should contain how to contribute to `compendium` which will include how to run `compendium` for development, using `sbt`.

Does it make sense to you @mrtmmr ?

Reply:
I meant not for contributing as developer but for using it. If we focus on make it work using docker image I would add a "Quick start"  section explaining requirements and how to run docker. It's not a big thing but I think it would make it more complete.

Reply:
Maybe I am missing something here, but, as I intended, in `docs/docs/docs/deployment/docker/index.md`, it explains how to use it with `docker` or using the `docker-compose` command. 

The requirements are explained here in the `docs/docs/docs/deployment/index.md`.

But I think it is out of the scope for compendium's documentation to explain how to set up a postgres for it, since there is _no_ proper way of doing it.

I agree that we should add a _Getting started_ section, which explains how to use `docker-compose` to run Compendium so anyone could run and play with it in less than a minute. But that should be done in another PR.

Let me know if I am missing something here :wink: